scripts/upgrade

```diff
@@ -101,30 +131,36 @@ then
 	sudo python $final_path/add_sso_conf_webdav.py
 	
 	# Update seafile config
-	echo 'COMPRESS_URL = MEDIA_URL' | sudo tee -a $final_path/seahub_settings.py
-	echo "STATIC_URL = MEDIA_URL + 'assets/'" | sudo tee -a $final_path/seahub_settings.py
+	echo 'COMPRESS_URL = MEDIA_URL' | sudo tee -a $final_path/conf/seahub_settings.py
+	echo "STATIC_URL = MEDIA_URL + 'assets/'" | sudo tee -a $final_path/conf/seahub_settings.py
 	
 	# Add logrotate
 	sudo cp ../conf/logrotate /etc/logrotate.d/seafile
 	sudo sed -i "s@SEAFILE_DIR@$final_path@g" /etc/logrotate.d/seafile
 	sudo service rsyslog restart
 fi
 
+# Update seahub config for old version to version 5.0.4
+if [[ $(sudo grep -c "LOGIN_URL" $final_path/conf/seahub_settings.py) == 0 ]]
+then
+        echo "LOGIN_URL = '"$path"/accounts/login/'" | sudo tee -a $final_path/conf/seahub_settings.py
```

Comment:
The $path variable is never set. During an upgrade I have this error:

2017-02-05 16:03:18,704 INFO     yunohost.hook <lambda> - [7846.1] + sudo tee -a /var/www/seafile/conf/seahub_settings.py
2017-02-05 16:03:18,716 WARNING  yunohost.hook <lambda> - [7846.1] ./upgrade: ligne 146: path : variable sans liaison

Strangely this doesn't cause the script to fail.